packages/common/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@sphereon/openid4vci-common",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "preinstall": "npx only-allow pnpm",
+    "clean": "rimraf dist coverage",
+    "build": "tsc -p ../../tsconfig.build.json",
+    "test": "echo \"INFO: no test specified\" && exit 0",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier \"{lib,tests}/**/*.ts\" --write",
+    "fix:lint": "eslint . --ext .ts --fix",
+    "uninstall": "rimraf dist coverage yarn.lock yarn-error.log package-lock.json node_modules"
+  },
+  "dependencies": {
+    "@sphereon/ssi-types": "^0.8.1-next.123"
+  },
+  "devDependencies": {
+    "rimraf": "^3.0.2",
+    "npm-run-all": "^4.1.5"
```

Comment:
Why is this dep here? It is not used in the scripts

Reply:
seemed like run-s was from this package.
it's clear to me now.
removed.